src/main/java/core/model/Model.java

```diff
@@ -200,38 +200,29 @@ public void setAnnotations(List<Annotation> annotations) {
      */
     public Boolean addCell(int id, String text, int nucleotides, CellType type) {
         switch (type) {
-            case RECTANGLE:
-                RectangleCell rectangleCell = new RectangleCell(id, nucleotides);
+            case RECTANGLE: RectangleCell rectangleCell = new RectangleCell(id, nucleotides);
```

Comment:
**NOTE:** Auto-format in IntelliJ IDEA will wreck this. Watch out with using it.